config/config.toml.example

```diff
@@ -215,9 +215,9 @@ bind-info-lease = "3s"
 
 # The limitation of the size in byte for the entries in one transaction.
 # If using TiKV as the storage, the entry represents a key/value pair.
-# WARNING: Do not set the value too large, otherwise it will make a very large impact on the TiKV cluster.
-# Please adjust this configuration carefully.
-txn-total-size-limit = 1073741824
+# NOTE: If binlog is enabled, this value should be less than 104857600 because the two configurations conflict with each other, otherwise this value is restrict by the RAM size.
```

Comment:
> The recommended value should be at most 1/6 of the RAM size, with limitations as following:
> * If binlog is enabled, this value should be less than 104857600(10M) because this is the maximum size that can be handled by Pumper.
> * If binlog is not enabled, this value should be less than 10737418240(10G).

How about this?

Reply:
Done